Sources/SymbolKit/SymbolGraph/Symbol/Symbol+KindIdentifier.swift

```diff
@@ -14,87 +14,61 @@ extension SymbolGraph.Symbol {
-            case .method: return "method"
-            case .property: return "property"
-            case .protocol: return "protocol"
-            case .struct: return "struct"
-            case .subscript: return "subscript"
-            case .typeMethod: return "type.method"
-            case .typeProperty: return "type.property"
-            case .typeSubscript: return "type.subscript"
-            case .typealias: return "typealias"
-            case .var: return "var"
-            case .module: return "module"
-            case .unknown: return "unknown"
-            }
+        private init(rawIdentifier: String) {
+            self.identifier = rawIdentifier
         }
 
-        // FIXME: Save "unknown" symbol kinds in a synchronized set to prevent loss of data (rdar://84276085)
+        public static let `associatedtype`  = KindIdentifier(rawIdentifier: "associatedtype")
+        public static let `class`           = KindIdentifier(rawIdentifier: "class")
+        public static let `deinit`          = KindIdentifier(rawIdentifier: "deinit")
+        public static let `enum`            = KindIdentifier(rawIdentifier: "enum")
+        public static let `case`            = KindIdentifier(rawIdentifier: "enum.case")
+        public static let `func`            = KindIdentifier(rawIdentifier: "func")
+        public static let `operator`        = KindIdentifier(rawIdentifier: "func.op")
+        public static let `init`            = KindIdentifier(rawIdentifier: "init")
+        public static let `method`          = KindIdentifier(rawIdentifier: "method")
+        public static let `property`        = KindIdentifier(rawIdentifier: "property")
+        public static let `protocol`        = KindIdentifier(rawIdentifier: "protocol")
+        public static let `struct`          = KindIdentifier(rawIdentifier: "struct")
+        public static let `subscript`       = KindIdentifier(rawIdentifier: "subscript")
+        public static let typeMethod        = KindIdentifier(rawIdentifier: "type.method")
+        public static let typeProperty      = KindIdentifier(rawIdentifier: "type.property")
+        public static let typeSubscript     = KindIdentifier(rawIdentifier: "type.subscript")
+        public static let `typealias`       = KindIdentifier(rawIdentifier: "typealias")
+        public static let `var`             = KindIdentifier(rawIdentifier: "var")
+        public static let module            = KindIdentifier(rawIdentifier: "module")
+
+        public static let allCases: [KindIdentifier] = [
+          .`associatedtype`,
+          .`class`,
+          .`deinit`,
+          .`enum`,
+          .`case`,
+          .`func`,
+          .`operator`,
+          .`init`,
+          .`method`,
+          .`property`,
+          .`protocol`,
+          .`struct`,
+          .`subscript`,
+          .`typeMethod`,
+          .`typeProperty`,
+          .`typeSubscript`,
+          .`typealias`,
+          .`var`,
+          .`module`
```

Comment:
Seems like we do not need the extra \`\` here except for .\`init\`

I think we should align with the Line24-42 code which mean either only add \`\` when necessary or add \`\` to all for neat.

```suggestion
            .associatedtype,
            .class,
            .deinit,
            .enum,
            .case,
            .func,
            .operator,
            .`init`,
            .method,
            .property,
            .protocol,
            .struct,
            .subscript,
            .typeMethod,
            .typeProperty,
            .typeSubscript,
            .typealias,
            .var,
            .module
```

Reply:
Also do not forget the `,` in the last case for `module`